oslabs-beta/KataLog
# MVP presenataion

##Overview
**Issue Type**
- [] Bug
- []Tech debt

**Description**
Finalized styling and certain functionality. Got the application up and running with all MVP features for a presentation. 

**Steps to Reproduce Bug/Validate Feature/Confirm Tech Debt Fix**
Go to the dashboard
Click on search or filter logs and see the logs render based on params
Scroll down to each individual log and click the arrow for more information

**Previous Behavior**
Styling and functionality wasn't up to date to encompass all MVP features.

**Expected Behavior**
MVP has been created and deployed. 

**How Has This Been Tested**
Tested through the UI and testing edge cases of parsing algos.

**Screenshots & Videos**
N/A

**Additional Context**
N/A
